tc/app/vault/get

```diff
@@ -30,23 +27,21 @@ function _vault_get() {
   export VAULT_TOKEN
   __secret_path=$1
   __env=$2
-  __output=$( vault kv get "$__secret_path" 2>&1 )
+  __project_dir=$(_get_project_path)
+  __output=$(vault kv get "$__secret_path" 2>&1)
   _error_code=$?
-  __MSG="Command: tcctl vault get $@ "
+  __MSG="Get ${__env} keys from ${__secret_path} - "
   __MSG+="$__output"
   if [ "$_error_code" -eq "0" ]; then
-    vault kv get "$__secret_path" | sed 's/\ \{1,\}/=/g'| sed 's:=n/a$:=:' | sed 1,11d > "$__env"
+    vault kv get "$__secret_path" | sed 's/\ \{1,\}/=/g'| sed 's:=n/a$:=:' | sed 1,11d > /tmp/"$__env"
+    if [[ -s /tmp/"$__env" ]]; then
+      mv /tmp/"$__env" "$__project_dir"/"$__env"
+    else
+      rm -rf /tmp/"$__env"
+    fi
```

Comment:
You need to take the path from the parameters, not define the static data. This is not the right decision.